retriever/lib/get_opts.py

```diff
@@ -59,9 +59,10 @@
 install_parser.add_argument('--debug', help='run in debug mode', action='store_true')
 install_parser.add_argument('--not-cached', help='overwrites local cache of raw data', action='store_true')
 download_parser.add_argument('dataset', help='dataset name').completer = ChoicesCompleter(script_list)
-ls_parser.add_argument('-l', help='verbose list of datasets containing following keywords '
+ls_parser.add_argument('-l', help='verbose list of datasets containing following keywords'
+                                  'or data sets with following license(s)'
                                   '(lists all when no keywords are specified)',
-                       nargs=1).completer = ChoicesCompleter(list(keywords_list))
+                       nargs='*').completer = ChoicesCompleter(list(keywords_list))
```

Comment:
`Retriever ls -l` this should be fixed here
When using the get_opts module the argument should not be `*`. argparse should be able to catch that and give you some warning that will almost be like `Retriever ls -l -h`

Reply:
yes I got it. So should I revert my last commit ?

Reply:
don't revert

Reply:
you can just keep committing. we shall do the cleaning up later. :+1:

Reply:
Fixed in this [commit](https://github.com/weecology/retriever/pull/1037/commits/3964f3babd15033c5633a1bc5044166801221512)